doc/guides/abstraction/posix.rst

```diff
@@ -0,0 +1,434 @@
+
+
+POSIX Support
+#############
+
+The Portable Operating System Interface (POSIX) is a family of standards
+specified by the IEEE Computer Society for maintaining compatibility between
+operating systems. Zephyr implements a subset of the embedded profiles PSE51
+and PSE52 and BSD Sockets.
+
+With the POSIX support available in Zephyr, an existing POSIX threading
+compliant application can be ported to run on the Zephyr kernel, and therefore
+leverage Zephyr features and functionality. Additionally, a library designed
+for use with POSIX threading compatible operating systems can be ported to
+Zephyr kernel based applications with minimal or no changes.
+
+.. image:: posix.svg
+   :align: center
+   :alt: POSIX Support in Zephyr
+
+Benefits of POSIX support in Zephyr:
+
+- Provide familiar API to non-embedded programmers, especially from Linux
+- Enable re-use (portability) of existing libraries based on POSIX APIs
+- Provide efficient subset appropriate for small (MCU) embedded systems
+- The POSIX API subset is an increasingly popular OSAL (operating system
+  abstraction layer) for IoT and embedded applications, as can be seen in:
+  - Zephyr
+  - AWS:FreeRTOS
+  - TI-RTOS
+  - NuttX
+
+
+System Overview
+===============
+
+Units of Functionality
+++++++++++++++++++++++
+
+The system profile is defined in terms of component profiles that specify Units
+of Functionality that can be combined to realize the application platform. A Unit
+of Functionality is a defined set of services which can be implemented. If
+implemented, the standard prescribes that all services in the Unit must
+be implemented.
+
+A Minimal Realtime System Profile implementation must support the
+following Units of Functionality as defined in IEEE Std. 1003.1 (also referred to
+as POSIX.1).
+
+
+.. csv-table:: Units of Functionality
+   :header: Requirements, Supported, Remarks
```

Comment:
maybe "Zephyr Supported" for the column heading in all these tables (to clarify)

Reply:
this is redundant, we are talking about zephyr here, not something else.